Cookbook/Technical-Documents/DebuggingPushNotifications.md

```diff
@@ -19,7 +19,7 @@ When you are testing push notifications make sure that:
 		- for production environment only use AppStore certificate (**or simply just never change production key set**)
 	- select proper [APNS environment](#apns-certificates-and-environments):
 		- for local builds (built from Xcode) select Development environment
-		- for Hockeyapp builds select Production environment
+		- for App Center builds select Production environment
 - make sure you notify in `#ios` slack channel that you are changing PubNub environment
 - REMEMBER to change APNS environment back to production when you are done and notify `#ios` slack channel. This is so that testing can continue on AppCentre builds.
```

Comment:
```suggestion
- REMEMBER to change APNS environment back to production when you are done and notify `#ios` slack channel. This is so that testing can continue on App Center builds.
```